xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/BlogParameters.xml

```diff
@@ -74,6 +74,6 @@
  * @param $imgName The name of the icon to use (without the extension).
  * @param alt The alternate text that should be displayed when images are not viewable.
  *#
-#macro(toolImage $imgName $alt)[[image:path:$xwiki.getSkinFile("icons/silk/${imgName}.png")||class="icon-manage" alt="${alt}"]]#end
+#macro(toolImage $imgName $alt)(% class="glyphicon glyphicon-${imgName} icon-manage" spam="${alt}"%) #end
```

Comment:
This doesn't look correct. IMO you shouldn't hardcode the icon set used!

See https://www.xwiki.org/xwiki/bin/view/FAQ/How+to+display+an+Icon+from+the+XWiki+Icon+Set